VCFv4.5.draft.tex

```diff
@@ -196,8 +196,56 @@ \subsubsection{Individual format field format}
   \item LR: Identical to R except the only alternate alleles defined in the $LAA$ field are considered present.
   \item LG: Identical to G except the only alternate alleles defined in the $LAA$ field are considered present.
   \item P: The field has one value for each allele value defined in $GT$.
+  \item M: The field has one value for each possible base modification for the corresponding ChEBI ID.
 \end{itemize}
 
+The cardinality of M fields is determined by genotype and number of possible base modifications for the corresponding alleles.
+The ID of all M fields must end with A, C, G, T, U, or N which defines the base(s) that the modification can occur on.
+U must be treated as synonymous with T.
+If any base modification key is present for a sample, GT must be defined for that sample.
+The number of base modification values for a given allele is the number of bases on either strand in the allele sequence that could contain the base modification.
+The order of the base modification values is the order that these bases occur in the allele.
+For N base modifications, the field contains values for both the positive and negative strands with the negative strand value immediately after the positive strand value.
+For example, an allele of CGA has 2 M5mC values, the first defining the methylation rate on forward strand C at the first base pair, and the second defining the methylation rate for reverse strand C at the second base pair.
+
+The order and number of alleles encoded in these fields is determined by the order and phasing in the genotype.
+Base modifications values are encoded in their GT order with one value for each possible base modification in the concatenated genotype allele bases.
+Unphased allele values are aggregated and encoded at the position of the first occurrence of the unphased allele value.
+MISSING allele values and symbolic alleles are treated as containing no relevant bases thus encode no base modification values.
+
+Unstranded base modification information should be stored at the base with the lowest POS with the other values MISSING.
+Unstranded N base modifications should be stored on the positive strand with the values MISSING.
+For example, unstranded 5mC CpG methylation should be stored on the VCF recording containing the C with the M5mC value of the subsequent G set to MISSING or omitted entirely. Similarly, unstranded MxaoN values should be stored in the positive strand value with the negative strand value MISSING.
+
+Examples: 
+
+\vspace{0.5em}
+\begin{tabular}{ l l l l l l l l l l}
+	\#CHROM & POS & REF & ALT & FORMAT & SAMPLE\\
+	chr & $10$ & C & A         & GT:M5mC       & \tt{0/1:0.95}\\
```

Comment:
Just reading this record, it is unknown whether these values are for all 5mC, just CpG methylation, and whether it is stranded or not.